modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -34,27 +38,32 @@ ifdef::google-cloud-platform[]
 
 You can use the `ccoctl gcp create-all` command to automate the creation of GCP resources.
 endif::google-cloud-platform[]
-ifdef::alibabacloud-default,alibabacloud-customizations[]
+ifdef::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 [id="cco-ccoctl-creating-at-once_{context}"]
 = Creating credentials for {product-title} components with the ccoctl tool
 
 You can use the {product-title} Cloud Credential Operator (CCO) utility to automate the creation of Alibaba Cloud RAM users and policies for each in-cluster component.
-endif::alibabacloud-default,alibabacloud-customizations[]
+endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 
 [NOTE]
 ====
 By default, `ccoctl` creates objects in the directory in which the commands are run. To create the objects in a different directory, use the `--output-dir` flag. This procedure uses `<path_to_ccoctl_output_dir>` to refer to this directory.
 ====
 
+[NOTE]
+====
+The `ccoctl` is a Linux binary that must run in a Linux environment.
+====
+
 .Prerequisites
 
 You must have:
 
 * Extracted and prepared the `ccoctl` binary.
```

Comment:
Suggest to tell a link on how to get `ccoctl`, e.g. https://docs.openshift.com/container-platform/4.12/authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.html#cco-ccoctl-configuring_cco-mode-gcp-workload-identity.

Reply:
@jianli-wei I have added some information on how to acquire the ccoctl binary. Thank you sir.